pyp5js/tests/test_config/test_sketch.py

```diff
@@ -0,0 +1,76 @@
+import json
+import os
+from pytest import fixture
+from pathlib import Path
+from tempfile import NamedTemporaryFile
+
+from pyp5js.config import TRANSCRYPT_INTERPRETER, PYODIDE_INTERPRETER
+from pyp5js.config.sketch import SketchConfig
+from pyp5js.config.fs import PYP5JS_FILES
+
+
+@fixture
+def transcrypt_json_file():
+    try:
+        fd = NamedTemporaryFile(mode='w', delete=False)
+        data = {"interpreter": "transcrypt"}
+        json.dump(data, fd)
+        filename = fd.name
+        fd.seek(0)
+        fd.close()
+        yield filename
+    finally:
+        os.remove(filename)
```

Comment:
I might be mistaken about this one, but I believe you could omit importing either Pathlib or os in this test. In this case, you can probably just use [Path.unlink](https://docs.python.org/3/library/pathlib.html#pathlib.Path.unlink) here.

```suggestion
        Path(filename).unlink()
```

Reply:
I think you're right @jvfe . The only reason we have `os.remove` instead of `Path.unlink` is because I did this and I'm not used to using pathlib... Hahahha
I agree it's better to use it.